### PR TITLE
#10. Make Column Config popup options configuration driven.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**

### CODE REVIEW

File: src/components/AppColumnConfigData.jsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
+export const columnsList = [
```
Comment (pepcus-pratik) [inline src/components/AppColumnConfigData.jsx]: This should be a JSON file.

File: src/components/ColumnConfig.jsx [hunk 2]
```diff
@@ -107,6 +136,7 @@ class ColumnConfig extends Component {
   };
 
   render() {
+    this.renderColumns();
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]: Is this required??

Comment (BhagyashreeBhoyar) [reply src/components/ColumnConfig.jsx]: no. I have removed it.

File: src/components/AppColumnConfigData.jsx (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/AppColumnConfigData.jsx]: Done.

File: src/components/ColumnConfig.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/ColumnConfig.jsx]: No, I have removed it.

**Pull request merged**